blog/_posts/btcpay-server-1-2-0.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "BTCPay Server 1.2.0 - Bootstrapping and paving roads"
+date: 2021-08-10
+author: Zaxounette
+category:
+  - "releases"
+tags:
+  - "btcpay-server"
+  - "release"
+  - "ui-ux"
+  - "greenfield"
+  - "api"
+  - "update"
+coverImage: "/images/BTCPayServer1.2.0.png"
+---
+
+It's been a few months since the last BTCPay Server major update and the team has been hard at work pushing through a number of efforts in that time, ranging from migrating our frontend UI framework to increasing again the Greenfield API capabilities, setting up the foundation for our plugin system and crushing numerous bugs!
+
+As the **BTCPay Server 1.2.0** major update includes more than forty items, we won't cover them all in this release post, but as usual the high-time preference crowd can view directly the [full changelog](https://github.com/btcpayserver/btcpayserver/releases).
+
+To update, simply navigate to your **Server Settings > Maintenance** tab and click **Update** or use the command *btcpay-update.sh* in the command line.
+
+
+## **Continuation of our UI updates**
+
+----
+
+Since the beginning of 2021, our Design prodigies at BTCPay Server have been refactoring and updating numerous pages inside the stack to make them less confusing, more comprehensible to a broader userbase and overall, easier to use.
+
+This release includes a number of these updated views, as well as the migration from Bootstrap3 to Bootstrap5. This framework migration enables us to create and implement more modern and user-friendly views.
+
+As examples, the *Crowdfund* and *Point-Of-Sale* creation pages have been updated to make it easier to navigate and more efficiently create your favorite Apps in BTCPay Server.
+
+![BTCPay 1.2.0 Crowdfund view](/images/BTCPay1.2.0Crowdfund.png)
+
+![BTCPay 1.2.0 POS view](/images/BTCPay1.2.0POS.png)
+
+A number of UI improvements and fixes also made it into this release, namely relating to:
+* The server settings view
+* The transaction signing view
+* The payment request view
+* The notifications view
+  
+For the full list, check out the [changelog](https://github.com/btcpayserver/btcpayserver/releases)
+
+## **BTCPay Plugins - Paving the road**
+
+----
+
```

Comment:
Probably a good idea to once again mention the vision for the plugins, just to give more context to the reader

Reply:
[43f2264](https://github.com/btcpayserver/blog/pull/9/commits/43f2264cef2b7c07cd44de094f8a76dcea38a16a)